src/renderer/features/wallet-details/model/wallet-proxies-model.ts

```diff
@@ -1,28 +1,34 @@
 import { type ApiPromise } from '@polkadot/api';
-import { combine } from 'effector';
+import { combine, createEvent, sample } from 'effector';
 import { createGate } from 'effector-react';
 
 import { proxyService } from '@/shared/api/proxy';
 import { type Chain, type ChainId, type Wallet } from '@/shared/core';
 import { type ProxyAccount } from '@/shared/core/types/proxy';
 import { createStoreFromEffect } from '@/shared/effector';
-import { entries, keys, nonNullable, nullable } from '@/shared/lib/utils';
-import { type AnyAccount, accountService, accounts } from '@/domains/network';
+import { keys, nonNullable } from '@/shared/lib/utils';
+import { type AnyAccount, accountService, accountSync, accounts } from '@/domains/network';
 import { networkModel, networkUtils } from '@/entities/network';
-import { accountUtils } from '@/entities/wallet';
+
+export type Proxy = Omit<ProxyAccount, 'id' | 'delay'> & { deposit: string };
+type WalletProxiesByChain = Record<ChainId, Proxy[]>;
```

Comment:
Looks like one issue still remains:
Deposit is not something that belongs to an individual proxy but rather is a single value for all proxies for the given account on the given chain.
Thus, I suggest to define types like the following
``` ts
type Proxy = Omit<ProxyAccount, 'id' | 'delay'>;
type ChainProxies = { proxies: Proxy[], deposit: string }
type WalletProxiesByChain = Record<Chain, ChainProxies>
```